src/data/sources.json

```diff
@@ -20,13 +20,13 @@
   {
     "name": "Metro UK",
     "site": "https://metro.co.uk/news/uk/",
-    "base": "",
+    "base": "https://12ft.io/https://metro.co.uk",
```

Comment:
The metro website is not behind a paywall, so doesn't need the addition of 12ft, but it may actually need the "https://www.metro.co.uk" as we're scraping a page beyond the homepage. Well spotted.